packages/composites/proof-of-reserves/src/endpoint/multiReserves.ts

```diff
@@ -0,0 +1,71 @@
+import type { ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
+import type { TInputParameters as SingleTInputParameters } from './reserves'
+import { execute as singleExecute } from './reserves'
+import { Validator } from '@chainlink/ea-bootstrap'
+import { Config } from '../config'
+import { AdapterError } from '@chainlink/ea-bootstrap'
+
+export const supportedEndpoints = ['multiReserves']
+
+export type TInputParameters = {
+  input: SingleTInputParameters[]
+}
+
+const inputParameters: InputParameters<TInputParameters> = {
+  input: {
+    required: true,
+    type: 'array',
+    description: 'An array of PoR request, each request is a request to the reserves endpoint',
+  },
+}
+
+export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
+  const validator = new Validator(input, inputParameters)
+  const jobRunID = validator.validated.id
+
+  const results = await Promise.all(
+    validator.validated.data.input.map((input) =>
+      singleExecute(
+        {
+          id: jobRunID,
+          data: input,
+        },
+        context,
+        config,
+      ),
+    ),
+  )
+
+  const result = results
+    .map((result) => {
+      if (result.statusCode != 200 || !result.result) {
```

Comment:
Nit: Could this be another 2XX code that is still successful?

Reply:
Looked into the reserves endpoint, looks like no